Source/WebCore/css/CSSSelector.h

```diff
@@ -67,7 +67,7 @@ struct PossiblyQuotedIdentifier {
         void replaceNestingParentByPseudoClassScope();
 
         // How the attribute value has to match. Default is Exact.
-        enum Match {
+        enum class Match {
```

Comment:
Could we hint at a size here?

`enum class Match : uint8_t {`

Reply:
I think Clang/LLVM defaults to 4 bytes, and that's clearly much larger than needed here.

Reply:
Yes indeed, I was hesitating to add an explicit size in this PR or another one. Let's do it in this one :)

Reply:
Changed ✅